vfox-restServer/src/main/java/com/vfoexchange/restServer/serviceImpl/EmailServicesImpl.java

```diff
@@ -20,20 +18,18 @@
     @Autowired
     private JavaMailSender emailSender;
 
-    private static Logger log = LoggerFactory.getLogger(EmailServicesImpl.class);
     @Override
     public void sendMail(final Mail mail) {
-        log.info("Mail - Sending  Email for verify ");
+        LOGGER.info("Sending  Email");
```

Comment:
We need to improve our logging to be more meaningful. Just "Sending Email" message doesn't help. Need more details. We can do this later.

Reply:
Added mail details in the LOGGER message, so that wherever sendMail() method will be called, it should print the mail details it is carrying.